src/main/java/pl/prawko/prawko_server/service/IRoleService.java

```diff
@@ -0,0 +1,18 @@
+package pl.prawko.prawko_server.service;
+
+import pl.prawko.prawko_server.model.Role;
+
+/**
+ * Service interface for managing {@link Role} entities.
+ */
+public interface IRoleService {
+
+    /**
+     * Find a matching role with provided role name.
+     *
+     * @param name role name that should be looked for
+     * @return matching role
+     */
+    Role findByName(final String name);
```

Comment:
I'd vote for `getRole`, as `findByName` seems like a JPA-related name that should be in a repository interface.